packages/okxwallet/README.md

```diff
@@ -0,0 +1,29 @@
+# @web3-onboard/okxwallet
+
+### Install
+
+`npm i @web3-onboard/okxwallet`
+
+## Usage
+
+```typescript
+import Onboard from '@web3-onboard/core'
+import okxWalletModule from '@web3-onboard/okxwallet'
+
+// initialize the module with options
+const okxWalletSdk = okxWalletModule({ darkMode: true })
```

Comment:
Please add available types for okx wallet init options

Reply:
Actually it doesnt look like there are available init options so those should be removed from this